client/lib/importer/test/api-interaction.js

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import Dispatcher from 'dispatcher';
+import nock from 'nock';
+import partial from 'lodash/function/partial';
+
+import { fetchState } from '../actions';
+import { actionTypes } from '../constants';
+import store from '../store';
+
+const fetchTestState = partial( fetchState, 0 );
```

Comment:
Not particularly concerned about it, but using a real site ID (e.g. `2916284`) could eliminate the chance for false positives. I might imagine the case where a condition for `if ( ! siteId ) {` could alter the outcome of the test.